app/templates/administrative-units/administrative-unit/local-involvements/edit.hbs

```diff
@@ -4,20 +4,27 @@
     <:title>Bewerk betrokken lokale besturen</:title>
     <:subtitle>{{@model.administrativeUnit.name}}</:subtitle>
     <:action>
-      <AuButton
-        @loading={{this.save.isRunning}}
-        @disabled={{this.save.isRunning}}
-        form="local-involvements-edit-creation"
-        type="submit"
-      >
-        Opslaan
-      </AuButton>
-      <AuLink
-        @route="administrative-units.administrative-unit.local-involvements"
-        @skin="button-secondary"
-      >
-        Annuleer
-      </AuLink>
+      <div class="au-o-flow au-d-component">
+        <div>
```

Comment:
Unrelated to this PR, but could you convert this to the `<AuButtonGroup>` component? That will add proper margin between buttons (at the moment the whitespace is only there because the template includes a space between the components).

I also noticed that the buttons aren't aligned to the right anymore:
![image](https://user-images.githubusercontent.com/3533236/145998714-aac87ad8-4b76-4c8b-8069-6a1bd6adfc1e.png)

Sadly the `<AuButtonGroup>` component [doesn't provide an argument for that yet](https://github.com/appuniversum/ember-appuniversum/issues/162) so we will need to add some custom css for now.

Maybe adding something like this to the _shame.scss file is good enough for now? 
```css
// Allow buttons to be aligned on the right side
.au-c-button-group--align-right {
  @include mq($from:medium) {
    justify-content: flex-end;
  }
}
```

Which can then simply be used like this:
```hbs
{{! TODO: Remove the custom class once the following issue is resolved: https://github.com/appuniversum/ember-appuniversum/issues/162 }}
<AuButtonGroup class="au-c-button-group--align-right">...
```

It's not perfect on all screen sizes but I think it's a good starting point.